### PR TITLE
159410: Initial updates to add the history tab

### PR DESCRIPTION
**Pull request by chrissmKainos**

### CODE REVIEW

File: app/api/applications.js [hunk 1]
```diff
@@ -74,9 +74,25 @@ async function processApplicationClaim (reference, user, approved) {
   }
 }
 
+async function getApplicationHistory (reference) {
+  const url = `${applicationApiUri}/application/history/${reference}`
+  try {
+    const response = await Wreck.get(url, { json: true })
+    if (response.res.statusCode !== 200) {
+      return { historyRecords: [] }
```
Comment (marcin-mogiela) [inline app/api/applications.js:82]: maybe worth logging what happened here + statusCode

**Review by marcin-mogiela — DISMISSED** [review]

File: app/api/applications.js (hunk 1 above)
Comment (chrissmKainos) [reply app/api/applications.js]: I've updated the code and test, please see latest commit.

**Review by marcin-mogiela — APPROVED** [review]

File: app/routes/reject-application-claim.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const Joi = require('joi')
 const { processApplicationClaim } = require('../api/applications')
+const getUser = require('../auth/get-user')
```
Comment (govTechSatish) [inline app/routes/reject-application-claim.js:3]: Can we put this in session once successfully logged in?

Comment (chrissmKainos) [reply app/routes/reject-application-claim.js:3]: @govTechSatish I guess we could but I'm not sure how you do this tbh. Can we get this merged so this isn't blocking testing and then maybe raise a task to add user to session?

Comment (govTechSatish) [reply app/routes/reject-application-claim.js:3]: Sure. Create small ticker to cover this backlog. Approving for now.

**Review by govTechSatish — APPROVED** [review]

**Pull request merged**